lib/kubernetes-deploy/renderer.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'securerandom'
+
+module KubernetesDeploy
+  class Renderer
+    def initialize(current_sha:, template_dir:, logger:, bindings: {})
+      @current_sha = current_sha
+      @template_dir = template_dir
+      @partials_dirs =
+        %w(partials ../partials).map { |d| File.expand_path(File.join(@template_dir, d)) }
+      @logger = logger
+      @bindings = bindings
+      # Max length of podname is only 63chars so try to save some room by truncating sha to 8 chars
+      @id = current_sha[0...8] + "-#{SecureRandom.hex(4)}" if current_sha
+    end
+
+    def template_variables
```

Comment:
Please make this method, `bind_template_variables` and `find_partial` private

Reply:
I don't think that improves anything.